js/src/logger.ts

```diff
@@ -304,17 +325,21 @@ class BraintrustState {
   }
 }
 
-let _state: BraintrustState;
+let _dangerousGlobalState: BraintrustState;
```

Comment:
personally I feel like calling it `_globalState` would be enough.

Reply:
changed